force-app/main/default/classes/ClassesObjectsMethods.cls

```diff
@@ -378,6 +444,14 @@ public with sharing class ClassesObjectsMethods {
      * @return The Person object of the older person.
      */
     public static Person getOlderPerson(Person person1, Person person2) {
-        return null; // Replace null with the variable you used to store the result
+        if(person1.getAge() > person2.getAge()){
+            return person1;
+        }
+        else if(person1.getAge() < person2.getAge()){
```

Comment:
Your second check here is probably redundant, The only options to return are person1 or person2. If you check one condition, you probably do not need to check the reverse.